packages/graphiql/src/components/DocExplorer/__tests__/FieldDoc.spec.js

```diff
@@ -35,47 +36,58 @@ const exampleObject = new GraphQLObjectType({
 
 describe('FieldDoc', () => {
   it('should render a simple string field', () => {
-    const W = mount(
+    const { container } = render(
       <FieldDoc
         field={exampleObject.getFields().string}
         onClickType={jest.fn()}
       />,
     );
-    expect(W.find('MarkdownContent').text()).toEqual('No Description\n');
-    expect(W.find('TypeLink').text()).toEqual('String');
-    expect(W.find('Argument').length).toEqual(0);
+    expect(container.querySelector('.doc-type-description')).toHaveTextContent(
+      'No Description',
+    );
+    expect(container.querySelector('.type-name')).toHaveTextContent('String');
+    expect(container.querySelector('.arg')).not.toBeInTheDocument();
   });
 
   it('should re-render on field change', () => {
-    const W = mount(
+    const { container, rerender } = render(
       <FieldDoc
         field={exampleObject.getFields().string}
         onClickType={jest.fn()}
       />,
     );
-    expect(W.find('MarkdownContent').text()).toEqual('No Description\n');
-    expect(W.find('TypeLink').text()).toEqual('String');
-    expect(W.find('Argument').length).toEqual(0);
+    expect(container.querySelector('.doc-type-description')).toHaveTextContent(
```

Comment:
I'm worried about how stable it will be using classes instead of component names, especially since we are about to change how layout/theming works and use theme-ui and @emotion

Reply:
hmm yeah, I wasn't to keen on using querySelector but I was trying to avoid making any changes to non-test code. Do you think it's worth adding either labels or testids to the components? We mostly just use a lot of classes in these components and react-testing-library doesn't let you target react implementation code